src/components/tech/JoinTeam.tsx

```diff
@@ -0,0 +1,57 @@
+import { Button, Flex, Heading, Section, Text, Link } from "@radix-ui/themes";
+import Image from "next/image";
+import React from "react";
+
+export const JoinTeam = () => {
+  return (
+    <Section
+      mx={{ md: "94px", initial: "2" }}
+      className="bg-[#98BEC6] rounded-3xl"
+      px={{ md: "9", initial: "6" }}
+      py="6"
+    >
+      <Flex
+        gap="6"
+        direction={{ lg: "row", initial: "column" }}
+        justify="center"
+        align="center"
+      >
+        <Flex gap="2" direction="column" justify="center">
+          <Heading size="8" className="text-navy-800">
+            How To Join Our Team!
+          </Heading>
+          <Heading size="5">
+            We’re usually looking to onboard new volunteers, especially
+            experienced backend and frontend developers.
+          </Heading>
+          <Text size="5">
+            You don’t need experience with our specific stack, as long as you’re
+            self-motivated and happy to expand your skills. It would be best if
+            you have at least some experience contributing to open-source
+            software. If you don’t yet, check out our [Open Source Explorers
+            program] instead. Our team meets once a month for a sprint meeting
+            to coordinate. We also host two casual weekly tech hangs on Zoom to
+            give folks a chance to collaborate.
+          </Text>
+          <Text size="4">
+            If you’re interested in joining our team, please tell us a bit about
+            yourself, why you want to join Distribute Aid and your skills. A
+            link to your GitHub profile would also be helpful!
+          </Text>
+
+          <Link href="mailto:tech@distributeaid.org">
+            <Button size="4" className="w-40">
```

Comment:
**nitpick:** Probably simpler to go with what Radix provides out of the box, and remove the width class.  Alternatively, you could wrap the button in a Box component with a set width, and use the `asChild` property to pass that down to the button.